test/packages/custom_logs/manifest.yml

```
@@ -0,0 +1,46 @@
+format_version: 1.0.0
+name: custom_logs
+title: Custom Logs
+description: >-
+  Collect custom logs with Elastic Agent.
```

We should add a requirement that category and description has to include the word "Custom", would be nice for filtering as well I believe?

---

I see the benefits of keeping the `custom` category and appropriate description, but not sure if this should be a hard requirement. I'd like to see comments from other folks.

---

For the category I don't have a strong opinion, but if required, maybe it'd be better to use an `input` category. For the description I would prefer to don't have any restriction. A common use case is to centralize logs without any parsing, and no "customization" applies there.

Looking to the historical background, Filebeat inputs are not described with the "custom" word. An input like this one could be described as in Filebeat:
```suggestion
  Read lines from active log files with Elastic Agent.
```